features/.useful_articles.md

```diff
@@ -0,0 +1,21 @@
+Cucumber anti-patterns (part one)
```

Comment:
Matt, thanks for reviewing.
I've combined .best_practices.md and .useful_articles.md into one file and moved it to doc folder, as well as .notes_and_tasks.md